app/controllers/katello/api/v2/systems_controller.rb

```diff
@@ -388,8 +388,13 @@ def setup_content_view(cv_id)
     end
 
     def systems_by_errata(errata_uuids, installable, non_installable)
-      installable = installable.nil? ? false : installable.to_bool
-      non_installable = non_installable.nil? ? false : non_installable.to_bool
+      if !!installable != installable
+        installable = installable.nil? ? false : installable.to_bool
+      end
+
+      if !!non_installable != non_installable
+        non_installable = non_installable.nil? ? false : non_installable.to_bool
+      end
```

Comment:
You lost me as I don't see any of those things here in this method other than the errata_uuids. If we want to check that `installable` or `non_installable` are booleans, why don't we just do that in the if-statement?

```
if installable == true
....
```